app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java

```diff
@@ -1471,7 +1470,10 @@ public void showLoading() {
             }
         }
 
-        PicassoHelper.cancelTag(PICASSO_VIDEO_DETAILS_TAG);
+        CoilUtils.dispose(binding.detailThumbnailImageView);
+        CoilUtils.dispose(binding.detailSubChannelThumbnailView);
+        CoilUtils.dispose(binding.overlayThumbnail);
```

Comment:
Aren't you missing `detailUploaderThumbnailView`?

Reply:
Thanks, I'll add that.